src/node_crypto.cc

```diff
@@ -1391,6 +1399,188 @@ void SecureContext::GetCertificate(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+#ifdef OPENSSL_PSK_SUPPORT
+
+void SecureContext::SetPskIdentity(const FunctionCallbackInfo<Value>& args) {
+  SecureContext* wrap = Unwrap<SecureContext>(args.Holder());
+
+  if (!args[0]->IsString()) {
+    return wrap->env()->ThrowTypeError("Argument must be a string");
+  }
```

Comment:
Not wrong but you might as well do the validation in JS land and simply `CHECK(args[0]->IsString())` here.

Reply:
Validation was already being done in JS, so this was definitely just redundant.